src/cmd/manage_nsfs.js

```diff
@@ -787,7 +789,39 @@ function validate_options(type, action, argv) {
         valid_options = VALID_OPTIONS.whitelist_options;
     }
     const invalid_input_options = input_options.filter(element => !valid_options.has(element));
-    return {invalid_input_options, valid_options};
+    if (invalid_input_options.length > 0) {
+        const type_and_action = type === TYPES.IP_WHITELIST ? type : `${type} ${action}`;
+        const invalid_option_msg = invalid_input_options.length === 1 ?
+        `${invalid_input_options[0]} is an invalid option` :
+        `${invalid_input_options.join(', ')} are invalid options`;
+        const supported_option_msg = `Supported options are: ${[...valid_options].join(', ')}`;
+        const err_msg = `${invalid_option_msg} for ${type_and_action}. ${supported_option_msg}`;
+        throw_cli_error(ManageCLIError.InvalidArgument, err_msg);
+    }
+}
+
+/**
+ *  validate_options_type check the type of the value that match what we expect
+ * @param {object} argv
+ */
+function validate_options_type(argv) {
+    const input_option_with_data = { ...argv };
+    delete input_option_with_data._;
+    for (const [option, value] of Object.entries(input_option_with_data)) {
```

Comment:
Do you mean in case the user didn't supply any option?
There are cases where there are no required flags, for example list:
- `sudo node src/cmd/manage_nsfs bucket list`
- `sudo node src/cmd/manage_nsfs account list`